prism/src/main/java/org/apache/falcon/util/SecureEmbeddedServer.java

```diff
@@ -48,6 +48,13 @@ protected Connector getConnector(int port) {
         connector.setPassword(properties.getProperty("password",
                 System.getProperty("password", "falcon-prism-passwd")));
         connector.setWantClientAuth(true);
+
+        // this is to enable large header sizes when Kerberos is enabled with AD
+        final Integer bufferSize = Integer.valueOf(StartupProperties.get().getProperty(
+                "falcon.jetty.request.buffer.size", "16192"));
```

Comment:
Good idea.   we have a bunch of these properties  - will create a separate JIRA to track that